CLR/Libraries/Windows_Devices/windows_devices_native_Windows_Devices_Spi_SpiBusInfo.cpp

```diff
@@ -0,0 +1,16 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include "windows_devices.h"
+
+HRESULT Library_windows_devices_native_Windows_Devices_Spi_SpiBusInfo::get_ChipSelectLineCount___I4( CLR_RT_StackFrame& stack )
+{
+    TINYCLR_HEADER();
+
+    CLR_RT_HeapBlock* pThis = stack.This(); FAULT_ON_NULL(pThis);
+
+    SetResult_INT32( stack, static_cast<INT32>(CPU_SPI_PortsCount()) );

```

Comment:
any reason why not:
stack.SetResult_I4( static_cast<INT32>(CPU_SPI_PortsCount())  ); 
?
generally speaking, the interop API was written for external customers that do not understand the CLR_RT_StackFrame object model.
a piece of small value when we were dealing with sub-par partners...